packages/cli/tests/e2e/bot/ProvisionResourceProjectWithNewBot.tests.ts

```diff
@@ -53,17 +60,34 @@ describe("Provision", function () {
     console.log(`[Successfully] provision for ${projectPath}`);
 
     {
-      // Validate provision
-      // Get context
-      const context = await readContext(projectPath);
-
-      // Validate Aad App
-      const aad = AadValidator.init(context, false, AppStudioLogin);
-      await AadValidator.validate(aad);
-
-      // Validate Bot Provision
-      const bot = BotValidator.init(context);
-      await BotValidator.validateProvision(bot);
+      if (isMultiEnvEnabled()) {
```

Comment:
Would it be better to separate multi-env cases from the original ones in a separated ts file? I mean, I saw if/else style here and concerned that can we cover both cases?

Reply:
In the end, normal case will be deleted. The only multi env case will be saved.